server/ntb/macros/nob_NO_translate_macro.py

```diff
@@ -56,13 +63,20 @@ def nob_NO_translate_macro(item, **kwargs):
         response = r.json()
         item.update(response["document"])
 
+        # restore exempted text by removing `<span lang="nb">` tags
+        for field, value in item.items():
+            if isinstance(value, str):
+                item[field] = re.sub(r"<span lang=\"nb\">(.*?)</span>", r"\1", value)
+
         # map translated `description_text` fields back to the associations.
         for editor in item.get("associations", {}):
             flat_key = f"associations_desc_{editor}"
             if flat_key in response["document"]:
-                item["associations"][editor]["description_text"] = response["document"][
-                    flat_key
-                ]
+                item["associations"][editor]["description_text"] = re.sub(
```

Comment:
would be good to avoid that duplication with `re.sub`, some helper method should do